src/main/java/net/tridentsdk/meta/item/ItemDisplayProperties.java

```diff
@@ -56,4 +56,30 @@
     default void setLore(String... lore) {
         setLore(Arrays.asList(lore));
     }
+    
+    /**
+     * Adds lore to the item starting from a specific index.
+     * @param index Index to begin at.
+     * @param lore Lore to be added.
+     * @see #addLore(String...)
+     */
+    default void addLore(int index, String... lore) {
+    	List<String> curr = lore();
+    	if (curr.isEmpty() || index == curr.size()) {
```

Comment:
You may also want to add that to the javadoc instead if it results in desired functionality